packages/inspection-capture-web/src/PhotoCapture/PhotoCaptureHUD/PhotoCaptureHUDTutorial/PhotoCaptureHUDTutorial.tsx

```diff
@@ -60,7 +60,7 @@ export function PhotoCaptureHUDTutorial({
               sightId={sightId}
               sightGuidelines={sightGuidelines}
               enableSightGuidelines={currentTutorialStep === TutorialSteps.GUIDELINE}
-              enableAddDamage={true}
+              addDamage={AddDamage.TWO_SHOT}
```

Comment:
Instead of passing `AddDamage.TWO_SHOT` to the `addDamage` prop, pass the real prop obtained from the `PhotoCapture` component